posix/linux.lisp

```diff
@@ -32,6 +32,19 @@
 (defsyscall "fdatasync" :int
   (fd file-descriptor-designator))
 
+(defsyscall ("mkstemps" %mkstemps) :int
+  (template :pointer)
+  (suffix-length :int))
+
+(defun mkstemps (template suffix)
+  (let ((actual-template (concatenate 'string
+                                      template
+                                      "XXXXXX"
+                                      suffix)))
+    (cffi:with-foreign-string (ptr-template actual-template)
+      (%mkstemps ptr-template (length suffix))
+      (cffi:foreign-string-to-lisp ptr-template))))
```

Comment:
The attraction of the `mkstemp*` functions is that they return an open file handle as well. Your wrapper should make sure to return that (look at the `mkstemp` wrapper).

Also, no need for the `cffi:` prefix. The `osicat-posix` package has `cffi` on the use list, for better or worse.

Reply:
Hi!

I think i have implemented your proposal, let me know if everything is fine or there is something other to fix.

Thanks again!
C.

Reply:
Cool. It also needs to be added to the list of unsupported functions in posix/windows.lisp. I'll also probably hold off on merging until I get a chance to get on my Darwin machine and check it there.

Reply:
Hi!

> Cool.

Thanks!

> It also needs to be added to the list of unsupported functions in posix/windows.lisp.

Sure, just added!

> I'll also probably hold off on merging until I get a chance to get on my Darwin machine and check it there.

No problem, take your time! :)
Bye!
C.